src/state-mutation.js

```diff
@@ -12,14 +12,16 @@ const updateReverseRelationship = (
   return (foreignEntities) => {
     return foreignEntities.update(
       foreignEntities.findIndex(
-        item => item.get('id') === relationship.getIn(['data', 'id'])
+        item => item.get('id') == relationship.getIn(['data', 'id'])
```

Comment:
My bad, I've been doing some research and I found out it was a bug inside one of the APIs I tried. Guess it will be better to revert this operator change.